tests/board_test.py

```diff
@@ -0,0 +1,36 @@
+from ..lib.board import *
+
+
+class TestBoard:
+    def test_move(self):
+        #     move method changes the value of the given key( number from the user) from empty to the given mark
+        new_board = Board()
+        new_board.move(1, "X")
+        assert new_board.board[1] == "X"
+
+    def test_position_taken(self):
+        # position_taken method will check if the value of the given key is not
```

Comment:
Same here, remove the comments and make the method name more descriptive. Also, I think it would be good to split these `assert`s into separate tests: one test that asserts that `position_taken` returns `True` and another for `False`